sdk/cosmos/azure-spring-data-cosmos-core/src/main/java/com/azure/spring/data/cosmos/repository/query/CosmosQueryMethod.java

```diff
@@ -27,6 +33,7 @@
      */
     public CosmosQueryMethod(Method method, RepositoryMetadata metadata, ProjectionFactory factory) {
         super(method, metadata, factory);
+        this.method = method;
```

Comment:
Can we just delete field `method`, use `stringInQueryAnnotation` instead?

Reply:
Refactored